Index: patroni/tests/test_citus.py
===================================================================
--- patroni.orig/tests/test_citus.py
+++ patroni/tests/test_citus.py
@@ -11,7 +11,10 @@ from patroni.psycopg import ProgrammingE
 from . import BaseTestPostgresql, MockCursor, psycopg_connect, SleepException
 from .test_ha import get_cluster_initialized_with_leader
 
+import unittest
 
+
+@unittest.skipIf(True, "Citus not tested")
 @patch('patroni.postgresql.mpp.citus.Thread', Mock())
 @patch('patroni.psycopg.connect', psycopg_connect)
 class TestCitus(BaseTestPostgresql):
